contrib/clojure-package/test/good-test-symbol.clj

```diff
@@ -1,9 +1,9 @@
 (ns org.apache.clojure-mxnet.symbol
-    (:refer-clojure :exclude [* - + > >= < <= / cast concat identity flatten load max
```

Comment:
Same here - This file should be excluded from the formatting - since the `test-write-to-file` uses it test whether the generator is working correctly and it makes the until test fail